### ISSUE TITLE
BUG? CLI options suddenly ignored

### ISSUE THREAD

**Issue by rngtng** [AnWeber/httpyac#477]
Hey. I think last release introduced a bug with cli options handling for `send`: The output format of response `-o <output>` doesn't an effect anymore (expect for `short`?). Same with passing list of variables `--var  <variables...>` - any value I set is not taken into account..

in short this command doesn't work as expected anymore:

```
httpyac send  -o none --var  token=foobar main.http
```

Thanks!

**Comment by AnWeber** [comment]
@rngtng I could understand the error with the output. But it only occurs with `none`, because here undefined was used and after the recent rebuild an empty object. However, using the variables worked for me.

```
GET https://httpbin.org/anything?token={{token}}
```
```sh
httpyac send headers.http --all --var token=foobar
```

**Comment by rngtng** [comment]
thanks @AnWeber for the quick fix. You're right the token set works. I figured, I used `token` with `{{$input-askonce token}}` and expected to be asked when `token` is already set via `--var`. I pretty sure this used to work. So I assumed the explicit var set is broken, but maybe it's related to `$input-askonce` then?

**Comment by AnWeber** [comment]
@rngtng I have added that input-askonce also reacts to already existing variables and takes them over directly if necessary. Both is released with 6.5.1

**Comment by rngtng** [comment]
Perfect. As always thanks a lot!!
